recipes/sensormapgeo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sensormapgeo" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sensormapgeo-{{ version }}.tar.gz
+  sha256: 4f435416cb98602882e0a122a5499b8ffb027a36b6eae5529e493ae235a5cfcd
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gdal
```

Comment:
Quick question. Do you know if this package needs the python bindings in `gdal`? If not you can depend on `libgdal` and make a slightly smaller package.

Reply:
Yes, the Python bindings are needed.